jscomp/core/lam_compile_external_call.ml

```diff
@@ -30,6 +30,10 @@ let splice_fn_apply fn args =
   E.runtime_call Js_runtime_modules.caml_splice_call "spliceApply"
     [ fn; E.array Immutable args ]
 
+let splice_fn_new_apply fn args =
+  E.runtime_call Js_runtime_modules.caml_splice_call "spliceNewApply"
+    [ fn; E.array Immutable args ]
+
 let splice_obj_fn_apply obj name args =
```

Comment:
I guess this should be called `splice_obj_apply`.

Reply:
Let's rename it then (it's not my code, though)